### ISSUE TITLE
How do you get the  "train_ben.pacp" and "test.pacp" accroding to kitsune dataset?

### ISSUE THREAD

**Issue by wuhustfu** [dongtsi/TrafficManipulator#7]

**Comment by dongtsi** [comment]
According to the paper of Kitsune, we use the first N (N=50k by default in Kitsune and our work) benign packets as the training set ("train_ben.pcap"). The remaining packets can be used as the testing set (in our work, we select fragments with as many malicious packets as possible as the test set/"test.pcap").
